nexus/tests/integration_tests/schema.rs

```diff
@@ -432,6 +436,16 @@ const CHECK_CONSTRAINTS: [&'static str; 4] = [
     "check_clause",
 ];
 
+const CONSTRAINT_COLUMN_USAGE: [&'static str; 7] = [
```

Comment:
Note: This wasn't used by the regression tests, but I figured I'd add it as "yet another check to validate that the DB schemas match".

See: https://www.cockroachlabs.com/docs/stable/information-schema#constraint_column_usage